components/Image.js

```diff
@@ -82,7 +83,7 @@ class Image extends PureComponent {
 }
 
 Image.propTypes = {
-  ...RNImage.propTypes,
+  ...ImagePropTypes.propTypes,
```

Comment:
I know this isn't from your previous PR, but this is an error, it's just supposed to be `...ImagePropTypes,`.